bears/vcs/git/GitCommitBear.py

```diff
@@ -31,6 +31,22 @@ class GitCommitBear(GlobalBear):
     }
     CONCATENATION_KEYWORDS = [r',', r'\sand\s']
 
+    _nltk_data_downloaded = False
+
+    def _check_imperative_enabled(self):
+        imperative_check = self.section.contents.get(
+            'shortlog_check_imperative', True)
+        return bool(imperative_check)
```

Comment:
I've removed the `.contents` , which makes this shorter and can be moved back into the `if` condition where it was being used.